src/components/Statistics/StatisticCharts.js

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { useState, useEffect, useMemo } from 'react';
+import {WeeklyStatistic} from './WeeklyReport'
+import {MonthlyStatistic} from './MonthlyReport'
+import { BackgroundCard, CustomWhiteCard, EventCard } from '../CustomMUIComponents/CustomCards';
+import PersistentDrawerLeft from "../NavDrawer/navDrawer";
+export default function StatisticCharts(){
+    const [show, setShow] = useState(true)
+
+    const StatisticComponent=(
```

Comment:
const StatisticComponent = React.useMemo( () => ( ...)).
This will memoize your component so it does not recompile it on every render.